packages/vue-use-tippy/source/index.ts

```diff
@@ -6,12 +6,13 @@ import {
 	onUnmounted,
 	watch,
 } from '@vue/composition-api'
+
 import castArray from 'lodash.castarray'
 import tippy, { Props, Instance } from 'tippy.js'
 
 type InstanceRefType = Instance<Props>[] | Instance<Props> | null
 type Callback = (t: InstanceRefType) => void
-type Target = Ref<Element | null> | string
+type Target = Ref<Element | null>
```

Comment:
string was removed from the union type because we use `Target[]` and tippy targets can't be `string[]`

so in order to be able to reuse `Target` as an array type, we split the `string` out of the union and support it directly on argument type

Reply:
this was previously a mistake, and thus is actually a FIX